01_introduction_to_algorithms/Purescript/packages.dhall

```diff
@@ -0,0 +1,128 @@
+{-
+Welcome to your new Dhall package-set!
```

Comment:
I'd remove these package set comments.
FYI, you can use `spago init -C` to omit these when creating a project.